lua/wikis/clashroyale/MatchSummary.lua

```diff
@@ -71,6 +71,7 @@ function CustomMatchSummary._createGame(game, gameIndex, date)
 			local playerCards = player.cards or {}
 			local cards = Array.map(Array.range(1, NUM_CARDS_PER_PLAYER), function(idx)
 				return playerCards[idx] or DEFAULT_CARD end)
+			---@cast cards table
```

Comment:
```suggestion
			---@cast cards string[]
```
Slightly more accurate I think?

Reply:
it is string[] at that point but need it as a table the line after so can insert the tower bullshit